deployer.tf

```
@@ -133,7 +135,18 @@ data "aws_iam_policy_document" "deployer-beanstalk" {
       "autoscaling:ResumeProcesses",
       "autoscaling:DescribeAutoScalingGroups",
       "autoscaling:DescribeLaunchConfigurations",
-      "autoscaling:PutNotificationConfiguration"
+      "autoscaling:PutNotificationConfiguration",
+      "elasticloadbalancing:DescribeLoadBalancers"
     ]
   }
+
+  statement {
+    sid       = "AllowLogsForBeanstalk"
+    effect    = "Allow"
+    actions   = [
+      "logs:CreateLogGroup",
+      "logs:PutRetentionPolicy"
+    ]
+    resources = ["arn:aws:logs:${data.aws_region.this.name}:*:log-group:/aws/elasticbeanstalk/${local.beanstalk_env}/*:*"]

```

I believe the trailing `:*` is unnecessary.
Also, I would reuse the local from log-reader:
```suggestion
    resources = ["arn:aws:logs:${data.aws_region.this.name}:*:log-group:${local.log_group}"]
```